prisma/schema.prisma

```diff
@@ -36,6 +37,21 @@ model Request {
   createdAt       DateTime          @default(now())
   updatedAt       DateTime          @updatedAt
   user            User              @relation(fields: [userId], references: [id]) // Specify fields and references here
+  Settlement      Settlement[]
+}
+
+model Settlement {
+  settlementId             Int                          @id @default(autoincrement())
+  requestId                Int                          @unique
+  userId                   Int
```

Comment:
add admin ID as mentioned in docs

Reply:
added the Dummy admin model and relation